imap_processing/tests/spice/test_data/imap_ultra_test_metakernel.template

```diff
@@ -0,0 +1,9 @@
+{SPICE_TEST_DATA_PATH}/imap_sclk_0000.tsc
```

Comment:
Great! I got this working with the new version of imap_wkcp.tf - I had been confused because it doesn't cover the same time ranges as the old euler file.